internal/engine/wazevo/testcases/testcases.go

```diff
@@ -1532,6 +1532,28 @@ var (
 			}}},
 		},
 	}
+
+	VecShuffle = TestCase{
+		Name:   "shuffle",
+		Module: VecShuffleWithLane(0, 1, 2, 3, 4, 5, 6, 7, 24, 25, 26, 27, 28, 29, 30, 31),
+	}
+
+	// VecShuffleWithLane returns a VecShuffle test with a custom 16-bytes immediate (lane indexes).
+	VecShuffleWithLane = func(lane ...byte) *wasm.Module {
```

Comment:
why is this a lambda function instead of a regular function?

Reply:
it was to keep it in the `var` block, but it sure can be a regular function.